crypto/nrf_cc310_platform/CHANGELOG.rst

```diff
@@ -9,6 +9,67 @@ Changelog - nrf_cc3xx_platform
 
 All notable changes to this project are documented in this file.
 
+nrf_cc3xx_platform - 0.9.16
+***************************
+
+New version of the library with the following features:
+
+* Added code-size optimized API for SHA-256 calculation in RAM and flash.
+* Fixed a bug where platform mutexes may end up unallocated if a context holding it is not zeroized.
+
+Library built against Mbed TLS version 3.1.0.
```

Comment:
```suggestion
```

This is not a change.

Reply:
It is not a change-statement.

Reply:
It's in the changelog.

Reply:
It is essential information, and this is a very accurate place for it. Won't change